spec/features/storing_files_spec.rb

```diff
@@ -20,6 +20,26 @@
     instance.file.delete
   end
 
+  it 'uploads a StringIO to the configured bucket' do
+    # https://github.com/carrierwaveuploader/carrierwave/wiki/How-to:-Upload-from-a-string-in-Rails-3-or-later
+    io = StringIO.new(image.read)
+
+    def io.original_filename
```

Comment:
I would rather us an expectation/stub than the straight def, but it is certainly clear.